apps/builder/app/builder/features/style-panel/sections/advanced/advanced.tsx

```diff
@@ -328,6 +336,79 @@ const $advancedProperties = computed(
   }
 );
 
+const AdvancedProperty = memo(
```

Comment:
Please consider making it reusable for not just advanced property, but for anything

Reply:
We don't need it anywhere else yet

Reply:
who will remember this solution even exists and where if its hidden in a single feature) but whatever, we tend to have copy pasted stuff like this a lot because we are too afraid to abstract too early